infra/modules/worklytics-connector-specs/main.tf

```
@@ -77,6 +77,45 @@ locals {
     import to Worklytics via this connection).
   2. Update the content of PSOXY_ASANA_ACCESS_TOKEN variable with the previous token value obtained
 EOT
+    }
+    cursor = {
+      source_kind : "cursor",
+      availability : "alpha",
+      enable_by_default : false,
+      worklytics_connector_id : "cursor-psoxy"
+      display_name : "Cursor"
+      worklytics_connector_name : "Cursor via Psoxy"
+      target_host : "api.cursor.com"
+      source_auth_strategy : "basic_auth" # cursor API uses basic auth (RFC 7617 Section 2, with API key as 'user-id' and no password
+      secured_variables : [
+        {
+          name : "BASIC_AUTH_USER_ID" # cursor's UX calls this an 'API Key', but it's actually a Basic Auth 'user-id'; should we have aliases or something?
+          writable : false
+          sensitive : true
+          value_managed_by_tf : false
+        }
+      ],
+      example_api_requests : [
+        {
+          method = "GET"
+          path   = "/teams/members"
+        },
+        {
+          method = "POST"
+          path   = "/teams/daily-usage-data"
+          # june 1st to july 1st, in epoch ms
+          body = "startDate=${timestamp("2025-06-01T00:00:00Z") * 1000}&endDate=${timestamp("2025-07-01T00:00:00Z") * 1000}"
```

This is not going to work, as [timestamp](https://developer.hashicorp.com/terraform/language/functions/timestamp) returns a string, not an object. I'd also put dynamic dates here -same for others-.

Dind't find a better proper function to convert a timestamp into unix time, so but I've found in the documentation is the usage of [time provider](https://registry.terraform.io/providers/hashicorp/time/latest/docs/functions/rfc3339_parse)

```suggestion
          body = "startDate=${provider::time::rfc3339_parse(timeadd(timestamp(), "-72h"))}&endDate=$${provider::time::rfc3339_parse(timeadd(timestamp(), "-24h"))}"
```